.gitignore

```diff
@@ -11,7 +11,8 @@
 *.pydev*
 *.vcproj
 .idea/
-uid.dll
+lib/uid
+lib/uid.exe
```

Comment:
If those files are not checked in, is the developer is required to manually download them?